pallets/hyperbridge_aggregations/src/lib.rs

```diff
@@ -110,11 +109,8 @@ pub mod pallet {
         aggregation: sp_core::H256,
     }
 
-    #[pallet::call]
     impl<T: Config> Pallet<T> {
         /// Dispatch aggregation to given EVM chain
-        #[pallet::weight(T::WeightInfo::dispatch_aggregation())]
-        #[pallet::call_index(0)]
         pub fn dispatch_aggregation(
             origin: OriginFor<T>,
             params: Params<T::Balance>,
```

Comment:
why the origin should be signed if it's just an internal call?

Reply:
Using a pallet account for this now